inf-terraform-aws/files/VERSION

```diff
@@ -0,0 +1 @@
+0.1.0
```

Comment:
Is this used by the quickstarter? If so, how does this correlate to version e.g. set in Jira?

Reply:
3fc73db